source/index.html.erb

```diff
@@ -113,22 +113,23 @@ Join Us
 
 		<div class="panel-body" id="events">
 
-			<h4>2016 CloudStack Collaboration Conferences</h4>
+			<h4>2017 CloudStack Collaboration Conferences</h4>
 
 			<p>
-			In 2016, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conferences</a> events will be held in the Americas. Each event will feature morning plenary sessions, afternoon breakout sessions in user and developer tracks and a hackathon. Learn more at: 
+			In 2017, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conference</a> events will be held in the Americas. Each event will feature a lot of opportunities to learn, collaborate and engage with the CloudStack community.  The CloudStack Collaboration Conferences are valuable for both developers and users, and are a great way to get involved with the community.
```

Comment:
I removed the Canada event and made sure that only Miami and Brazil showed on that site now.  Thanks...